### PR TITLE
Add id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418

### PR DESCRIPTION
**Pull request by glimow**
Adding waren's identifier-splitter-nn model.

### CODE REVIEW

File: id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# id_splitter_nn
+
+Model that contains source code identifier splitter BiLSTM weights.
+
+Example:
+
+```python
+from sourced.ml.models.id_splitter import IdentifierSplitterNN
+id_splitter = IdentifierSplitterNN().load("522bdd11-d1fa-49dd-9e51-87c529283418")
```
Comment (irinakhismatullina) [inline id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: Why didn't you add the backend argument in the example?

Comment (glimow) [reply id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: because I completely forgot, mostly.
duh, sorry

Comment (glimow) [reply id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: Can this be edited ?

Comment (irinakhismatullina) [reply id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: I'm not sure, the thing is that the meta is saved to the `index.json` automatically. Probably if you just edit it in both places it's ok, idk. I'd force-publish it again myself, but idk what's the right way here.

Comment (glimow) [reply id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: @irinakhismatullina fixed

Comment (irinakhismatullina) [reply id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: Awesome, thank you!

**Comment by irinakhismatullina** [comment]
@vmarkovtsev PTAL

**Review by zurk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -52,6 +52,23 @@ print("Number of tokens:", len(id2vec))
 *  [3467e9ca-ec11-444a-ba27-9fa55f5ee6c1](/id2vec/3467e9ca-ec11-444a-ba27-9fa55f5ee6c1.md)
 * <default> [92609e70-f79c-46b5-8419-55726e873cfc](/id2vec/92609e70-f79c-46b5-8419-55726e873cfc.md)
 
+## id_splitter_nn
+
+Model that contains source code identifier splitter BiLSTM weights.
```
Comment (vmarkovtsev) [inline README.md]: Should be (change the code in ml accordingly):

```
Weights of the BiLSTM network to split source code identifiers.
```

File: id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+```python
+from sourced.ml.models.id_splitter import IdentifierSplitterNN
+import modelforge.backends
+id_splitter = IdentifierSplitterNN().load("522bdd11-d1fa-49dd-9e51-87c529283418", backend=modelforge.backends.create_backend())
+id_splitter.split(identifiers)
+```
+
+### References
+
+- []()
```
Comment (vmarkovtsev) [inline id_splitter_nn/522bdd11-d1fa-49dd-9e51-87c529283418.md]: There should be a reference to https://arxiv.org/abs/1805.11651

File: index.json [hunk 4]
```diff
@@ -1 +1 @@
...
..."idna", "2.8"], ["jsonschema", "2.6.0"], ["modelforge", "0.11.1"], ["numpy", "1.16.2"], ["pandas", "0.24.2"], ["pip", "19.0.3"], ["protobuf", "3.7.1"], ["pygtrie", "2.3"], ["python-dateutil", "2.8.0"], ["pytz", "2019.1"], ["requests", "2.21.0"], ["scipy", "1.2.1"], ["semantic-version", "2.6.0"], ["setuptools", "41.0.0"], ["six", "1.12.0"], ["sourced-ml", "0.8.3"], ["spdx", "2.5.0"], ["tensorflow-estimator", "1.13.0"], ["tensorflow-gpu", "1.13.1"], ["termcolor", "1.1.0"], ["urllib3", "1.24.1"]], "platform": "Linux-4.15.0-47-generic-x86_64-with-debian-stretch-sid", "python": "3.6.0 (default, Jan  7 2019, 11:18:05) [GCC 5.4.0 20160609]"}, "license": "ODbL-1.0", "metrics": {}, "references": [[]], "series": "rnn-splitter", "tags": ["identifiers_splitter"], "vendor": "source{d}", "version": [1, 0, 0], "code": "from sourced.ml.models.id_splitter import IdentifierSplitterNN\nimport modelforge.backends\nid_splitter = IdentifierSplitterNN().load(%s, backend=modelforge.backends.create_backend())\nid_splitter.split(identifiers)", "extra": {}, "size": "8.7 MB", "url": "https://storage.googleapis.com/models.cdn.sourced.tech/models%2Fid_splitter_nn%2F522bdd11-d1fa-49dd-9e51-87c529283418.asdf"}}}}
```
Comment (vmarkovtsev) [inline index.json]: I know that what I will ask is not really related, but we really need to pretty-format JSON here.

This means editing https://github.com/src-d/modelforge/blob/master/modelforge/index.py#L202 (**and maybe other places**) to supply `sort_keys=True, indent=4`

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: index.json (hunk 4 above)
Comment (glimow) [reply index.json]: https://github.com/src-d/modelforge/pull/91

File: README.md (hunk 2 above)
Comment (glimow) [reply README.md]: Done in ML too

File: index.json (hunk 4 above)
Comment (glimow) [reply index.json]: @vmarkovtsev PTAL

**Comment by glimow** [comment]
Updated to match comments @vmarkovtsev

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]
Two things left:

- Regenerate the JSON
- Commit message style

**Pull request closed without merging**